src/main/scala/com/github/sbt/git/ConsoleGitRunner.scala

```diff
@@ -16,9 +16,11 @@ object ConsoleGitRunner extends GitRunner {
   private lazy val cmd = if(isWindowsShell) Seq("cmd", "/c", "git") else Seq("git")
 
   // in order to enable colors we trick git into thinking we're a pager, because it already knows we're not a tty
-  val colorSupport: Seq[(String, String)] =
-    if(ConsoleLogger.formatEnabled) Seq("GIT_PAGER_IN_USE" -> "1")
+  val colorSupport: Seq[(String, String)] = {
+    import sbt.internal.util.Terminal.console
+    if(console.isAnsiSupported && console.isColorEnabled) Seq("GIT_PAGER_IN_USE" -> "1")
```

Comment:
I don't think it is safe to use `Terminal.console` because it seems [here](https://github.com/sbt/sbt/blob/8ead89691f9466cba698c989b4f4573897203f48/internal/util-logging/src/main/scala/sbt/internal/util/Terminal.scala#L383-L422) that it can stay uninitialized forever.

My suggestion is:
- to continue using the deprecated `ConsoleLogger.formatEnabled`
- or to catch [this `IllegalStateException`](https://github.com/sbt/sbt/blob/8ead89691f9466cba698c989b4f4573897203f48/internal/util-logging/src/main/scala/sbt/internal/util/Terminal.scala#L847) and return false